public/app/plugins/datasource/grafana-testdata-datasource/dataquery.gen.ts

```diff
@@ -1,12 +1,4 @@
-// Code generated - EDITING IS FUTILE. DO NOT EDIT.
-//
-// Generated by:
-//     public/app/plugins/gen.go
-// Using jennies:
-//     TSTypesJenny
-//     PluginTsTypesJenny
-//
-// Run 'make gen-cue' from repository root to regenerate.
+// Code was originally generated cue
```

Comment:
From now on it's manual?

Reply:
yup -- I updated the file names to make that more clear.

We have a good path for jsonschema >> typescript generation; but not wired up yet